RUFAS/data_collection_app_updater.py

```diff
@@ -25,6 +29,15 @@
 """Fallback placeholder in a DCA input field if no value has been entered into it."""
 INPUT_PLACEHOLDER: str = "null"
 
+"""Directory to the available user feed inputs."""
+USER_FEED_PATH: str = os.path.join(os.path.dirname(__file__), "..", "input", "data", "feed", "user_feeds.csv")
+
+"""Path to all the DCA scripts"""
+SCRIPT_PATH = os.path.dirname(os.path.abspath(__file__))
+
+"""Path to the feed_schema.js schema file."""
+FEED_SCHEMA_PATH = os.path.join(SCRIPT_PATH, "..", "DataCollectionApp", "schema", "feed_schema.js")
```

Comment:
```suggestion
"""Directory to the available user feed inputs."""
USER_FEED_PATH: Path = Path("").joinpath("input", "data", "feed", "user_feeds.csv")

"""Path to the feed_schema.js schema file."""
FEED_SCHEMA_PATH: Path = Path("").joinpath("DataCollectionApp", "schema", "feed_schema.js")
```
Using paths instead of strings didn't seem to break the code when I tried it out. I think it would be better to do it like this because it keeps it consistent with how the other module constant Paths are defined.

Also I recommend moving these up to be with the other module constants that end with `_PATH`, so that the paths and placeholders remain distinctly grouped.

Might also be able to get rid of the `os` import.